keystrokes/src/components/Character/CharacterDetails/KeybindTable.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import {
+	Grid,
+	TableContainer,
+	Table,
+	TableHead,
+	TableRow,
+	TableCell,
+	TableBody,
+} from '@material-ui/core';
+import PropTypes from 'prop-types';
+import Paper from '@material-ui/core/Paper';
+import IconButton from '@material-ui/core/IconButton';
+import styleGuide from '../../../stylesheets/style';
+import EditIcon from '@material-ui/icons/Edit';
+import DeleteIcon from '@material-ui/icons/Delete';
+
+KeybindTable.propTypes = {
+	allKeybinds: PropTypes.array.isRequired,
+};
+
+function KeybindTable({ allKeybinds }) {
+	const classes = styleGuide();
+	return (
+		<Grid item>
+			<TableContainer component={Paper}>
+				<Table
+					className={classes.table}
+					size="small"
+					aria-label="a dense table"
+				>
+					<TableHead>
+						<TableRow>
+							<TableCell>Spell</TableCell>
+							<TableCell align="right">Target</TableCell>
+							<TableCell align="right">Modifier</TableCell>
+							<TableCell align="right">Key</TableCell>
+							<TableCell align="right"></TableCell>
+						</TableRow>
+					</TableHead>
+					<TableBody>
+						{allKeybinds.map(row => (
+							<TableRow key={row.Spell + row.Target}>
```

Comment:
looks like a few references to row.Prop
could deconstruct the properties from row

Reply:
i think ill be passing row in the future to something, so ill just keep it like this for now. To do that though, would i just make it so that its something like `row => { {Spell, Target, Mod, Key} = row return (<TableRow...)`? or can you change the row call at the start of the map?